packages/mwp-app-route-plugin/src/route.js

```diff
@@ -12,7 +15,13 @@ export const onPreResponse = {
 	 */
 	method: (request: HapiRequest, reply: HapiReply) => {
 		const response = request.response;
-
+		if (request.query) {
+			const trackingParam = request.query['_xtd']
```

Comment:
linter should probably be set up to catch this, but you can use dot notation `request.query._xtd`